api/swagger.yml

```diff
@@ -4993,6 +5002,51 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /repositories/{repository}/branches/{branch}/objects/stat/user_metadata:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: branch
+        required: true
+        schema:
+          type: string
+        description: branch to update
+      - in: query
+        name: path
+        description: path to object relative to the branch
+        required: true
+        schema:
+          type: string
+    put:
+      tags:
+        - objects
+        - experimental
+      operationId: updateObjectUserMetadata
+      summary: update object metadata
```

Comment:
Maybe we should mention this will override the current metadata

Reply:
Done, thanks!